tests/J0030+0451_prof_pre.txt

```diff
@@ -0,0 +1,256 @@
+0.00000       1.90144
```

Comment:
Is there actual test code that uses these files?

Reply:
I do not think the files are used in test code. Should I go ahead and delete them?

Reply:
Please don't add files that are never used. (This PR seems to be adding that file.)